### PR TITLE
dev-cpp/ETL: EAPI7, add missing die, use HTTPs

### PR DESCRIPTION
**Pull request by mm1ke**
Hi,

This PR raises EAPI, adds a missing ```die``` and uses https instead of http. Also dropped the *DEPEND vars.
I didn't introduced a new revision as nothing changed installation wise and the package isn't stable anyway.

Please review.

### CODE REVIEW

**Comment by mm1ke** [comment]
Ping, just rebased this PR, any chance for a merge?

**Comment by monsieurp** [comment]
@lu-zero: ping?

File: dev-cpp/ETL/ETL-0.04.15.ebuild [hunk 1]
```diff
@@ -1,26 +1,24 @@
...
 
 DESCRIPTION="ETL is a multi-platform class and template library"
-HOMEPAGE="http://synfig.org"
+HOMEPAGE="https://www.synfig.org"
 SRC_URI="mirror://sourceforge/synfig/${PN}/${PV}/${P}.tar.gz"
 
 LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
 IUSE=""
```
Comment (ktrace) [inline dev-cpp/ETL/ETL-0.04.15.ebuild]: empty IUSE may be deleted

**Comment by mm1ke** [comment]
ping, just rebased this PR

**Comment by mm1ke** [comment]
@monsieurp 
Ok, no problem. I've updated this PR and added a revbump for ETL.

**Comment by mm1ke** [comment]
ping, any chance we can merge this :)

**Comment by mm1ke** [comment]
another ping, would be nice if we could merge this :)

**Pull request closed without merging**